keith-vscode/src/pasta/stpa-interaction.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ *
+ * Copyright 2023 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import * as vscode from 'vscode';
+import { handleWorkSpaceEdit } from '../util';
+
+/** Command identifiers that are provided by pasta. */
+const pastaCommands = {
+    getLTL: 'pasta.getLTLFormula',
+    // sendModelCheckerResult: 'pasta.sendModelCheckerResult'
+};
+
+/**
+ * Registers commands that interact with PASTA
+ * @param context The extension context.
+ */
+export function registerStpaCommands(context: vscode.ExtensionContext): void {
+    context.subscriptions.push(
+        vscode.commands.registerCommand('keith-vscode.import-stpa-ltl', async (...commandArgs: any[]) => {
+            importStpaLTL(commandArgs[0]);
+        })
+    );
+}
+
+/**
+* Imports LTL formulas into the scchart given by {@code currentUri}. 
+* @param currentUri The uri of the scchart in which the LTL should be imported.
+*/
+async function importStpaLTL(currentUri: vscode.Uri): Promise<void> {
+    const pastaExtension = vscode.extensions.getExtension('kieler.pasta');
+    if (pastaExtension) {
```

Comment:
A little cleaner way to write this with less indentation of the code is to use the "fail fast" approach, checking `!pastaExtension` here , showing the error and immideately returning, thus not requiring any else with a big code block.